bin/update-package-references

```diff
@@ -19,11 +19,16 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 cd $DIR/..
 
-lastCommitted=$(git log --pretty=oneline --abbrev-commit | grep "Updated package.json references to artifacts" | head -n 1 | cut -d' ' -f1)
+git log --pretty=oneline --abbrev-commit | tail -n 10
```

Comment:
This logic is supposed to shortcircuit the work if no changes to `etna/` were found in the commits since the last `etna reference updates` commit.  It works, sometimes, but I've noticed it being over eager.  I've added these additional lines as a means to help debug that issue and plan to remove them when I've got this nailed down.